custom_components/tuya_local/devices/geyserwise_mws_water_heater.yaml

```diff
@@ -0,0 +1,217 @@
+name: Geyserwise MWS
+products:
+  - id: qwbyia2yr5ip9y4v
+    manufacturer: Geyserwise
+    model: MWS
+    name: Solar MPPT Geyser
+entities:
+  - entity: water_heater
+    dps:
+      - id: 1
+        type: boolean
+        name: operation_mode
+        mapping:
+          - dps_val: false
+            value: "Off"
```

Comment:
Values should be valid operation modes. Lower case "off", electric, heat_pump etc (see https://developers.home-assistant.io/docs/core/entity/water-heater/#states for full list)